html/js-array-map.html

```diff
@@ -0,0 +1,24 @@
+<html>
+
+<body>
+    <p id="demo"></p>
+    <script>
+        var firstValue = [1, 5, 7, 4, 98, 8]
+        var secondValue = firstValue.map(raiseToPower);
+        // var selection=firstValue.filter(biggerThan);
+
+        document.getElementById("demo").innerHTML = secondValue;
+        // document.getElementById("demo").innerHTML = selection;
+
+
+        function raiseToPower(value, index, array) {
```

Comment:
The function name does not completely represent what the function does.

Suggestion
- change the name of the function with a more specific one